.eslintrc

```diff
@@ -3,7 +3,8 @@
   "extends": [
 		"airbnb",
 		"plugin:react/recommended",
-		"plugin:@next/next/recommended"
+		"plugin:@next/next/recommended",
+		"plugin:storybook/recommended"
```

Comment:
I think the order here matters so I would put `"plugin:storybook/recommended"` right after `"airbnb"`.

Reply:
It does, that's true. We shouldn't overwrite current rules